package.json

```diff
@@ -32,7 +32,7 @@
     "test": "playwright test --config=tests/library/playwright.config.ts",
     "eslint": "eslint --cache",
     "tsc": "tsc -p . && tsc -p packages/html-reporter/",
-    "build-installer": "babel -s --extensions \".ts\" --out-dir packages/playwright-core/lib/utils/ packages/playwright-core/src/utils",
+    "build-installer": "esbuild packages/playwright-core/src/utils/**/*.ts --outdir=packages/playwright-core/lib/utils --platform=node --format=cjs --sourcemap=linked",
```

Comment:
I don't see this script being used anywhere, perhaps just remove it? I don't even think it works.

Reply:
Yes, I was not sure either, decided to keep to be on the safe side. But since we all agree I'll drop it.